### DISCUSSION TITLE
Permanent 1.25 /5/2.0 x video speed option

### DISCUSSION THREAD

**Discussion by gototheskinny** [dayanch96/BHTikTok-Plus#15] [Ideas]
Not for just 1 video but whole session or permanent.
It would be AWESOME. Thank you for this tweak!

**Comment by dayanch96** [comment]
Anlayamadım kanka, her zaman seçtiğin bir hızda oynanmasını mı istiyorsun?

**Reply by gototheskinny** [reply]
> Anlayamadım kanka, her zaman seçtiğin bir hızda oynanmasını mı istiyorsun?

Evet mesela 2x i seçtim diyelim her zaman 2x oynatma hızı kalsın. Ben değiştirene kadar tabi. Bunun aynısı Androidde var, plugin modda. Mümkünmü bilmiyorum ama yapsan çok iyi olurdu. Bu arada bu tweak müthiş gerçekten eline sağlık lrd den kurtardın bizi.

**Reply by dayanch96** [reply]
Eyw. Bakacağım eğer olursa eklerim güncellemeyle
